### DISCUSSION TITLE
Overseerr Series Requests

### DISCUSSION THREAD

**Discussion by I-am-PUID-0** [itsToggle/plex_debrid#265] [Q&A] [answered]
Hi @itsToggle 

How are tv series requests from overseerr handled? Namely, does it continue to search for new episodes as they're released as with the watchlist or is it a singular event that searches for all episodes up to the date of the request?

THX

**Comment by itsToggle** [comment, marked as answer]
Hi, plex_debrid should continue looking for new episodes of requested series, same as anything you add to your watchlist. The only difference in overseerr is that you can also specify which seasons you want to download

**Reply by I-am-PUID-0** [reply]
Awesome! Thanks again!
